packages/platform/atoms/event-types/wrappers/EventTypeWebWrapper.tsx

```diff
@@ -20,6 +20,7 @@ import { HttpError } from "@calcom/lib/http-error";
 import { telemetryEventTypes, useTelemetry } from "@calcom/lib/telemetry";
 import { SchedulingType } from "@calcom/prisma/enums";
 import { trpc, TRPCClientError } from "@calcom/trpc/react";
+import useMeQuery from "@calcom/trpc/react/hooks/useMeQuery";
```

Comment:
skipped the import on https://github.com/calcom/cal.com/pull/16808#discussion_r1780564577
causing the type error